### ISSUE TITLE
The differences between dfrft(x,1) and fft(x)

### ISSUE THREAD

**Issue by JiacLuo** [tunakasif/torch-frft#14]
# Question:

Hello，dear author.
         I have a question about **the different values of Fast Fourier Transform and Discrete Fractional Fourier Transform (alpha=1). According to relevant theory, these two values ​​should be the same**, but in the experiment the results are different, as shown in the following code and running results.
        However, when I drew the images of the two, I found that their sequence shapes were the same, except that the coordinate values were different, as shown in the figure below.
        Since I don't know much about the signal processing field, I kindly ask you to tell me why this happens.Looking forward to your answer very much！
                                                                                              

# Code:
```python
import torch
from torch.fft import fft, ifft, fftshift
from torch_frft.dfrft_module import dfrft
import matplotlib.pyplot as plt
n = 20
torch.manual_seed(0)
x = torch.rand(n)
y1 = dfrft(x,1)
y2 = fft(x)
print(y1-y2)
plt.plot(torch.arange(n),y1,'s-',color = 'r',label="frft")
plt.plot(torch.arange(n),y2,'o-',color = 'g',label="fft")
plt.xlabel("time")
plt.ylabel("value")
plt.legend(loc = "best")
plt.show()
```

# Running results：

```python
y1:
tensor([ 1.9222-3.1278e-07j, -0.0071-9.2283e-02j, -0.1030+4.4788e-01j,
         0.0392-2.1012e-01j,  0.2214-2.9596e-01j,  0.1335-1.7273e-01j,
         0.1406-1.7950e-01j, -0.0218-5.6817e-03j, -0.0657+7.8567e-02j,
         0.0209-7.9984e-02j, -0.4188+4.1989e-07j,  0.0209+7.9984e-02j,
        -0.0657-7.8567e-02j, -0.0218+5.6815e-03j,  0.1406+1.7950e-01j,
         0.1335+1.7273e-01j,  0.2214+2.9596e-01j,  0.0392+2.1012e-01j,
        -0.1030-4.4788e-01j, -0.0071+9.2283e-02j])
y2:
tensor([ 8.5961+0.0000j, -0.0318-0.4127j, -0.4608+2.0030j,  0.1755-0.9397j,
         0.9903-1.3236j,  0.5969-0.7725j,  0.6286-0.8027j, -0.0973-0.0254j,
        -0.2940+0.3514j,  0.0935-0.3577j, -1.8728+0.0000j,  0.0935+0.3577j,
        -0.2940-0.3514j, -0.0973+0.0254j,  0.6286+0.8027j,  0.5969+0.7725j,
         0.9903+1.3236j,  0.1755+0.9397j, -0.4608-2.0030j, -0.0318+0.4127j])
y1-y2:
tensor([-6.6740-3.1278e-07j,  0.0247+3.2042e-01j,  0.3578-1.5551e+00j,
        -0.1363+7.2958e-01j, -0.7689+1.0276e+00j, -0.4634+5.9976e-01j,
        -0.4880+6.2325e-01j,  0.0756+1.9728e-02j,  0.2283-2.7279e-01j,
        -0.0726+2.7772e-01j,  1.4540+4.1989e-07j, -0.0726-2.7772e-01j,
         0.2283+2.7279e-01j,  0.0756-1.9728e-02j, -0.4880-6.2325e-01j,
        -0.4634-5.9976e-01j, -0.7689-1.0276e+00j, -0.1363-7.2958e-01j,
         0.3578+1.5551e+00j,  0.0247-3.2042e-01j])
```

# Figure

- dfrft:
![image](https://github.com/tunakasif/torch-frft/assets/156330040/b7946af4-0dfc-4c0e-808e-520b3d94de8d)

- fft:
![image](https://github.com/tunakasif/torch-frft/assets/156330040/16e9e4e2-c0c6-421d-82a4-ea23c3cffdae)

- dfrft and fft in the same drawing:
![image](https://github.com/tunakasif/torch-frft/assets/156330040/4fd3072c-bb8d-46dd-8a42-28c5cf2dc198)

**Comment by tunakasif** [comment]
Two values should not be the same; the Discrete Fractional Fourier Transform is a unitary transformation for all transform orders, including $a=1$. FFT is not a unitary transformation, so you need to use the normalized (orthonormal) FFT.

This means that `dfrft(x, 1)` is not equal to `fft(x)` **but to `fft(x, norm="ortho")`**. Please refer to the papers in the [`README.md`](https://github.com/tunakasif/torch-frft/blob/main/README.md) for the theory and [PyTorch documentation](https://pytorch.org/docs/stable/generated/torch.fft.fft.html) for FFT implementation details.

Also, beware of the floating point equalities. Since we are also dealing with complex floats, it is expected to see value differences in the range of $[10^{-8}, 10^{-6}]$. To check equality, consider using a tolerance value according to your input size. Examples can be found in the repository tests.

**Comment by JiacLuo** [comment]
Thank you very much for your answer！
